third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -30,7 +31,8 @@ namespace {
 enum class state_type
 {
     WAIT_FOR_DEVICE_HEADER,
-    WAIT_FOR_PROFILES,
+    WAIT_FOR_DEVICE_OPTIONS,
+    WAIT_FOR_STREAMS_INFO,
```

Comment:
INFO?

Reply:
I suggest splitting into `HEADER` and `OPTIONS`, same as for device

Reply:
`stream-header`s and `stream-options` are interleaved. Did not want to force sender for send all streams than all options order.
Can also remove the states completely and just wait for all info or timeout...

Reply:
They can be interleaved, no problem.

Reply:
Done